articles/cosmos-db/partitioning-overview.md

```diff
@@ -102,6 +102,9 @@ For **all** containers, your partition key should:
 
 If you need [multi-item ACID transactions](database-transactions-optimistic-concurrency.md#multi-item-transactions) in Azure Cosmos DB, you will need to use [stored procedures or triggers](how-to-write-stored-procedures-triggers-udfs.md#stored-procedures). All JavaScript-based stored procedures and triggers are scoped to a single logical partition.
 
+> [!NOTE]
+> In case you only have one physical partition, partition key may not be relevant as all queries will hit the same physical partition. 
+
```

Comment:
```suggestion
> [!NOTE]
> If youu only have one physical partition, the value of the partition key may not be relevant as all queries will target the same physical partition. 
```

Just some quick editing, otherwise this note is great!